### PR TITLE
kickoff

### PR DESCRIPTION
**Pull request by TomasTomecek**
Signed-off-by: Tomas Tomecek <ttomecek@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
-# workshop
-This is a workshop to teach you modules packaging.
+# Modularity workshop
+
+## High-level agenda
+
+1. 1-minute-intro to modularity
+
+2. `modulemd.yaml`
+ * Guidelines
```
Comment (TomasTomecek) [inline README.md:8]: @james-antill will you be working on finishing guidelines this sprint?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+ * Best practices (do we have this? is this guidelines?)
```
Comment (TomasTomecek) [inline README.md:9]: @asamalik @nphilipp @KarstenHopp what's your opinion here?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+ * Buildtime dependencies
+   * List of well-known modules (What is the status of platform module? I don't want to teach base-runtime)
```
Comment (TomasTomecek) [inline README.md:11]: @contyk hello

File: README.md [hunk 4]
```diff
@@ -1,2 +1,30 @@
...
+   * What components are part of a specific module? (What is the status of this tooling?)
```
Comment (TomasTomecek) [inline README.md:12]: @asamalik @KarstenHopp can I get your help here?

Comment (KarstenHopp) [reply README.md:12]: Adam is working on a list of components of each module, have a look at https://github.com/modularity-modules. Unfortunately no CLI tool yet, afaik.

**Comment by contyk** [comment]
I have no background on this. Is it a Flock workshop?

File: README.md (hunk 3 above)
Comment (contyk) [reply README.md:11]: Platform is an ever evolving module/concept. The status depends on when you intend to deliver this workshop. At the moment there are no Platform builds.

**Comment by TomasTomecek** [comment]
@contyk yes, this will be a flock workshop + we'll do some trial rounds for interested packagers before the actual flock conference.

Edit: And the question I have for you: will we be able to teach using platform module instead of base runtime (~2 weeks timeframe)?

File: README.md (hunk 4 above)
Comment (TomasTomecek) [reply README.md:12]: Unfortunately I don't see any listings there.

**Comment by contyk** [comment]
@TomasTomecek Yes, definitely.

File: README.md (hunk 4 above)
Comment (KarstenHopp) [reply README.md:12]: As I said, he's working on it during this Sprint, Trello #1428 . https://github.com/modularity-modules/python2 shows an example of what he's planning to do.

File: README.md (hunk 3 above)
Comment (TomasTomecek) [reply README.md:11]: We agreed that we should deliver the workshop by August 9th.

File: README.md (hunk 4 above)
Comment (TomasTomecek) [reply README.md:12]: Right! Now I understand, thank you!

**Pull request merged**